streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -411,25 +384,22 @@ public void seek(final TopicPartition partition, final long offset) {
 
         shouldNotSeek.set(new AssertionError("Should not seek"));
 
+        @SuppressWarnings("unchecked")
         final java.util.function.Consumer<Set<TopicPartition>> resetter =
-            EasyMock.mock(java.util.function.Consumer.class);
-        resetter.accept(Collections.singleton(partition1));
-        EasyMock.expectLastCall();
-        EasyMock.replay(resetter);
+            mock(java.util.function.Consumer.class);
+        doNothing().when(resetter).accept(Collections.singleton(partition1));
```

Comment:
Yeah, that is the main reason why I kept it as a doNothing rather than verification. This being said, I like your solution because it is explicit and in my opinion it is a stronger assertion than relying on StrictStubs, so I will use that one